api/end_device.proto

```diff
@@ -775,7 +775,15 @@ message EndDevice {
   // This field is set by the Application Server and stored in the Identity Server.
   google.protobuf.Timestamp last_seen_at = 54;
 
-  // next: 55;
+  // VendorID managed by the LoRa Alliance, as defined in TR005.
```

Comment:
I think it would be better if these two identifiers (`vendor_id` and `vendor_profile_id`) become part of a common `TR005Identifiers` message, which would be a `tr005_identifiers` field of `EndDevice` instead.

@johanstokking thoughts on this ?

Reply:
Hmm they are introduced in TR005 but these are general LoRa Alliance primary identifiers. So I'd rather see `lora_alliance_vendor_id` then. Good idea indeed to avoid confusion.

Reply:
@johanstokking Can I please get some clarification on this? Are you suggesting that the `vendor_id` and `vendor_profile_id` be prefixed with `lora_alliance_` and remain in the top level of `EndDevice`, or should they be prefixed and remain as part of the `version_ids`?